src/components/Navigation/Navigation.scss

```diff
@@ -102,133 +70,63 @@
     .section-number {
       color: inherit;
     }
-
-    .eapp-status-icon-error {
-      opacity: 1;
-    }
   }
 
-  .is-valid {
-    .section-number {
-      color: $eapp-green !important;
-    }
+  .section-link {
+    // override USWDS
+    background: none;
+    font-weight: normal;
 
-    .eapp-status-icon-valid {
-      opacity: 1;
-    }
+    position: relative;
   }
 
-  .section-name {
-    display: inline-block;
-    max-width: 87.6%;
-    color: $eapp-grey-dark;
+  // override USWDS
+  .usa-accordion > li {
+    background: none;
   }
-
-  .section-number {
-    position: absolute;
-    left: 0.4rem;
-    display: block;
-    width: 2.5rem;
-    font-weight: $eapp-bold;
-    text-align: right;
-    color: #6c7178;
+  .usa-accordion-content {
+    padding: 0;
   }
 
-  // Animations
-  .section,
-  .subsection {
-    .section-content {
-      animation-duration: 0.5s;
-      animation-fill-mode: forwards;
-      animation-iteration-count: 1;
-      overflow: hidden;
-    }
-
-    &.open .section-content {
-      animation-name: section-openbox;
-      opacity: 1;
-      transition: opacity 300ms ease-out;
-    }
+  // subnavs
+  .usa-sidenav-sub_list {
+    .section-link {
+      border-left: $sidenav-current-border-width solid transparent;
```

Comment:
I don't think this value is defined anywhere. I was getting an error when building locally

Reply:
Try https://github.com/18F/e-QIP-prototype/pull/482#issuecomment-403553284.